ThemeSetup.php

```diff
@@ -171,9 +152,6 @@ public static function init_gutenberg_config() {
 
 		// Wide/Fullwidth images.
 		add_theme_support( 'align-wide' );
-
-		// Responsive embeds.
-		add_theme_support( 'responsive-embeds' );
```

Comment:
@honekun why did you remove responsive-embeds support? I think we should preserve it.

Reply:
Agree, I'll add it back